### PR TITLE
perf(engine): perf improvements in release-notes

### PR DESCRIPTION
**Pull request by tbouffard**
Insist on Engine startup and BDM services resume in the release notes
page to highlight this point for end-user.

Relates to [BS-11460](https://bonitasoft.atlassian.net/browse/BS-11460)

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,22 @@ We have forgotten that in 7.5
...
+A major improvement has been integrated in Bonita 7.6.0, especially for tenants with a lot of processes, processes with large dependencies or processes with a
+lot of dependencies
+
+Tests conducted both internally at Bonitasoft and on customer sites with real production data have shown the following decreases
+* execution time by a factor of 5 to 10
+* JVM Heap memory consumption by a factor of 2 to 3
+
+### Living Application
+
+To be documented
```
Comment (cpuy) [inline md/release-notes.md:27]: I would rather document it or remove it

**Comment by cpuy** [comment]
LGTM appart from the TODO part :-)

Moreover, it would be easier for us to read if you configure your IDE with 120 as line width ;-)

File: md/release-notes.md (hunk 1 above)
Comment (tbouffard) [reply md/release-notes.md:27]: It is a reminder to ensure we won't miss it during the release notes final review
It should be done through [BS-17011](https://bonitasoft.atlassian.net/browse/BS-17011)

**Comment by tbouffard** [comment]
@cpuy thanks for your review
About the 120 line width: I did not find any information in the CONTRIBUTING.md file of the current repository. So I use a 160 line width configuration as for other Bonita projects.

We should definitely agree on the width value for doc md files and document it in the contributing guideline. 120 or 160 is ok for me.

File: md/release-notes.md (hunk 1 above)
Comment (cpuy) [reply md/release-notes.md:27]: Ok then I guess we can let it as it is and document it later if it is what you want

**Comment by cpuy** [comment]
Actually we changed the formatter for java source code https://github.com/bonitasoft/bonita-developer-resources/pull/3

You're right there is no strict rule for md files but I believe this is a good general rule to limit to 120 for most files

**Comment by tbouffard** [comment]
Ok, I didn't notice this change and I guess I am not the only one :-)

**Pull request merged**